.gitpod.dockerfile

```diff
@@ -19,5 +19,7 @@ RUN bash -c ". .nvm/nvm.sh \
     && nvm install $NODE_VERSION \
     && nvm use $NODE_VERSION \
     && nvm alias default $NODE_VERSION \
+    && npm config set python /usr/bin/python --global \
+    && npm config set python /usr/bin/python \
     && npm install -g yarn"
```

Comment:
Gitpod's default image also installs `typescript` here -- not needed for Theia?

https://github.com/gitpod-io/workspace-images/blob/23a8109ab27b1355421477366657ce85b54b8043/full/Dockerfile#L196

Reply:
we already have typescript in Theia package.json that should do